src/buildercore/cfngen.py

```diff
@@ -217,7 +217,10 @@ def build_context_rds(context, existing_context):
 
     # pull password from existing context, if it exists
     generated_password = utils.random_alphanumeric(length=32)
-    rds_password = existing_context.get('rds_password', generated_password)
+    rds_password = existing_context.get('rds_password')
+    if not rds_password:
+        # may be present but None
```

Comment:
`existing_context.get('rds_password') or generated_password`

Reply:
I wonder if there is a familiarity vs readability argument that could be made here ...

Reply:
Was a quick hack to make it work, I think I'll go through this module in a follow-up